tools/upgrade_versions.py

```diff
@@ -225,7 +225,7 @@ def push_packages_head_commit(
 
 def run(CONF):
     versions_repo = setup_versions_repository(CONF)
-    packages_to_update = CONF.get('common').get('packages') or PACKAGES
+    packages_to_update = PACKAGES
```

Comment:
1. It is faster. Not every test during development needs to be complete.
2. Sometimes I need to clean the environment to test a new feature, sometimes the paths change (gyr's latest patches), sometimes I just don't want to wait to download the last week's of kernels updates.